wallet/chain/p/wallet/backend_visitor.go

```diff
@@ -63,18 +67,47 @@ func (b *backendVisitor) RemoveSubnetValidatorTx(tx *txs.RemoveSubnetValidatorTx
 }
 
 func (b *backendVisitor) TransferSubnetOwnershipTx(tx *txs.TransferSubnetOwnershipTx) error {
-	b.b.setSubnetOwner(
+	b.b.setOwner(
 		tx.Subnet,
 		tx.Owner,
 	)
 	return b.baseTx(&tx.BaseTx)
 }
 
 func (b *backendVisitor) ConvertSubnetTx(tx *txs.ConvertSubnetTx) error {
+	for i, vdr := range tx.Validators {
+		b.b.setOwner(
+			tx.Subnet.Prefix(uint64(i)),
```

Comment:
Should this be Append? (also should we add a name like `validationID` for this?)

Reply:
Yes - this is incorrect.

Reply:
I'm also a bit confused with Prefix. I also thought it should be Append.

Reply:
Updated